src/Storages/MergeTree/ReplicatedMergeTreeRestartingThread.cpp

```diff
@@ -347,8 +339,6 @@ void ReplicatedMergeTreeRestartingThread::partialShutdown(bool part_of_full_shut
     storage.partial_shutdown_event.set();
     storage.replica_is_active_node = nullptr;
 
-    LOG_TRACE(log, "Waiting for threads to finish");
```

Comment:
These messages were useful sometimes

Reply:
It was a mistake on my part. I had added a bunch of extra logs to help me understand what was going on and deleted some that I didn't add.